pkg/crc/machine/driver_darwin.go

```diff
@@ -39,9 +30,6 @@ func getDriverOptions(machineConfig config.MachineConfig) interface{} {
 	// Supported drivers
 	switch machineConfig.VMDriver {
 
-	case "virtualbox":
-		logging.Warn("Virtualbox support is deprecated and will be removed in the next release.")
```

Comment:
Maybe we want to keep special casing "virtualbox" for a while?
```
case "virtualbox":
    errors.ExitWithMessage(1, "Virtualbox support has been removed, use the native hyperkit driver instead. If you cannot do without vbox, please file a bug so that your usecase can be considered")
```
(message is just a suggestion :)

Reply:
@code-ready/crc-devel WDYT?

Reply:
perhaps moving this to earlier, as part of the pre-flight would make more sense... else this will fail during start (which is actually too late in the process, as it gices the expectation it might work).

Reply:
If you update this message as @cfergeau suggested then you also need to exit asap instead calling the `CreateHost` otherwise it should go in preflight check but then it is more work since you have to provide again the driver name to preflight functions which recently removed by 92f0496 one.

Reply:
The full code for that function is
```
func getDriverOptions(machineConfig config.MachineConfig) interface{} {
	var driver interface{}

	// Supported drivers
	switch machineConfig.VMDriver {

	case "hyperkit":
		driver = hyperkit.CreateHost(machineConfig)

	default:
		errors.ExitWithMessage(1, "Unsupported driver: %s", machineConfig.VMDriver)
	}

	return driver
}
```
I don't think we should go for something very sophisticated and exit as soon as possible in preflights, I'm simply suggesting to have a more detailed message in the virtualbox case rather than exiting with a generic `Unsupported driver` message.

Reply:
@cfergeau Right so what you are suggesting something like? which is good for us since we are using `errors.ExitWithMessage` so exiting asap.

```golang
func getDriverOptions(machineConfig config.MachineConfig) interface{} {
	var driver interface{}

	// Supported drivers
	switch machineConfig.VMDriver {

	case "hyperkit":
		driver = hyperkit.CreateHost(machineConfig)
        case "virtualbox":	
		errors.ExitWithMessage(1, "Virtualbox support has been removed, use the native hyperkit driver instead. If you cannot do without vbox, please file a bug so that your usecase can be considered")
	default:
		errors.ExitWithMessage(1, "Unsupported driver: %s", machineConfig.VMDriver)
	}
	return driver
}
```

Reply:
> @cfergeau Right so what you are suggesting something like? which is good for us since we are using `errors.ExitWithMessage` so exiting asap.

Yes, that's it! And that's really just a minor suggestion, the generic message is ok too.

Reply:
@gbraad Can you make this change? We should have this in before this release.

Reply:
I've pushed https://github.com/cfergeau/crc/tree/remove-vbox which has these changes.

Reply:
After further discussion/experiments, these changes are not necessarily useful as the invalid machine driver name will be caught by validation.go

Reply:
> case "virtualbox":
    errors.ExitWithMessage(1, "Virtualbox support has been removed, use the native hyperkit driver instead. If you cannot do without vbox, please file a bug so that your usecase can be considered")

Reconsideration is not possible